ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 5.4.0b14-HTP
+m_EditorVersion: 5.4.0b18-HTP
```

Comment:
Looks like B22 is actually the latest now, but this probably needs to be done separately in another PR.